e2e/cli-e2e/tests/print-config.e2e.test.ts

```diff
@@ -11,8 +11,8 @@ describe('CLI print-config', () => {
   it.each(extensions)(
     'should load .%s config file with correct arguments',
     async ext => {
-      const { code, stderr, stdout } = await executeProcess({
-        command: 'npx',
+      const { code, stdout } = await executeProcess({
+        command: 'code-pushup',
```

Comment:
Is this change correct with `'@code-pushup/cli'` kept in args?

Reply:
it was trying to use the new setup also in the cli-e2e, but there where too many edge cases.
Now all reverted.